include/opendht/dht_proxy_server.h

```diff
@@ -237,6 +238,9 @@ class OPENDHT_PUBLIC DhtProxyServer
     std::thread schedulerThread_;
 
     Sp<Scheduler::Job> printStatsJob_;
+    Sp<Scheduler::Job> getNodeStatsJob_;
```

Comment:
merge with printStatsJob to ensure stats are printed after the update (and not maybe just before)

Reply:
printStatsJob_ print only subscribers and listeners, I don't think it's necessary? But yeah we can only use one job

Reply:
Indeed it's different stats. Might still optionally be merged in a single job.

Reply:
Done